src/ipx.ts

```diff
@@ -27,7 +28,7 @@ export const ipxSetup: IPXSetupT = setupOptions => (providerOptions, moduleOptio
   }
 
   // Options
-  const absoluteDir = resolve(nuxt.options.srcDir, moduleOptions.dir || nuxt.options.dir.public)
+  const absoluteDir = await getDevDirs(nuxt, moduleOptions)
```

Comment:
nit pick: `absoluteDirs` or `publicDirs` or `publicDevDirs` could be better new names for this variable.